plugins/cql-language/src/test/java/org/polypheny/db/cql/utils/CqlInterfaceTest.java

```diff
@@ -77,89 +92,108 @@ public void testRestCqlFiltersOnlyQueryWithOROperator() {
+                    "test.employee.deptno":5,"test.dept.deptname":"Accounting and Finance"},
+                   \s
+                    {"test.employee.married":false,"test.employee.empname":"Anagha","test.dept.deptno":5,"test.employee.empno":18,
+                    "test.employee.joining_date":"2021-10-14","test.employee.dob":"2000-10-14","test.employee.salary":90000.0,
+                    "test.employee.deptno":5,"test.dept.deptname":"Accounting and Finance"},
+                   \s
+                    {"test.employee.married":true,"test.employee.empname":"Diana","test.dept.deptno":5,"test.employee.empno":19,
+                    "test.employee.joining_date":"1970-06-03","test.employee.dob":"1945-01-03","test.employee.salary":19700.0,
+                    "test.employee.deptno":5,"test.dept.deptname":"Accounting and Finance"},
+                   \s
+                    {"test.employee.married":false,"test.employee.empname":"Matt","test.dept.deptno":5,"test.employee.empno":20,
+                    "test.employee.joining_date":"1989-06-16","test.employee.dob":"1956-12-13","test.employee.salary":33000.0,
+                    "test.employee.deptno":5,"test.dept.deptname":"Accounting and Finance"},
+                   \s
+                    {"test.employee.married":true,"test.employee.empname":"Holt","test.dept.deptno":6,"test.employee.empno":21,
+                    "test.employee.joining_date":"1990-01-01","test.employee.dob":"1958-02-25","test.employee.salary":87000.0,
+                    "test.employee.deptno":6,"test.dept.deptname":"IT"},
+                   \s
+                    {"test.employee.married":false,"test.employee.empname":"Peralta","test.dept.deptno":6,"test.employee.empno":22,
+                    "test.employee.joining_date":"1990-01-01","test.employee.dob":"1980-01-31","test.employee.salary":22000.0,
+                    "test.employee.deptno":6,"test.dept.deptname":"IT"},
+                   \s
+                    {"test.employee.married":true,"test.employee.empname":"Mando","test.dept.deptno":6,"test.employee.empno":23,
+                    "test.employee.joining_date":"1990-01-01","test.employee.dob":"1967-11-30","test.employee.salary":35700.0,
+                    "test.employee.deptno":6,"test.dept.deptname":"IT"},
+                   \s
+                    {"test.employee.married":false,"test.employee.empname":"Vader","test.dept.deptno":6,"test.employee.empno":24,
+                    "test.employee.joining_date":"1990-01-01","test.employee.dob":"1959-12-13","test.employee.salary":3000.0,
+                    "test.employee.deptno":6,"test.dept.deptname":"IT"}],
+                   \s
+                    "size":20}
+                """ );
+
+        cqlInterfaceTestHelper( "test.employee.empno >= 1 NOT test.dept.deptname = \"Human Resources\"", expectedJsonNode, false );
     }
 
 
     @Test
-    @Ignore
+    @Disabled
```

Comment:
Can this test be enabled?

Reply:
Adjusted with changes in https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1521142328, enabled.